karabo_bridge/simulation.py

```diff
@@ -38,39 +38,45 @@ class Detector:
     pixel_size = 0  # [mm]
     distance = 0  # Sample to detector distance [mm]
     layout = np.array([[]])  # super module layout of the detector
-    const_fields = {}
 
     @staticmethod
-    def getDetector(detector, source='', corr=True, gen='random'):
+    def getDetector(detector, source='', raw=False, gen='random'):
         if detector == 'AGIPD':
-            source = source or 'SPB_DET_AGIPD1M-1/DET/detector'
-            return AGIPD(source, corr=corr, gen=gen)
+            if not raw:
+                default = 'SPB_DET_AGIPD1M-1/CAL/APPEND_CORRECTED'
+            else:
+                default = 'SPB_DET_AGIPD1M-1/CAL/APPEND_RAW'
+            source = source or default
+            return AGIPD(source, raw=raw, gen=gen)
         elif detector == 'AGIPDModule':
-            source = source or 'SPB_DET_AGIPD1M-1/DET/0CH0:xtdf'
-            return AGIPDModule(source, corr=corr, gen=gen)
+            source = source or 'SPB_DET_AGIPD1M-1/CAL/0CH0:xtdf'
```

Comment:
This name probably also differs for raw data.

Reply:
I've just double checked that device name should read 'SPB_DET_AGIPD1M-1/DET/0CH0:xtdf' instead of 'SPB_DET_AGIPD1M-1/CAL/0CH0:xtdf'

Reply:
Possibly the source name is different for calibrated data?

Reply:
Yes they should be different. I've decided to raise an error if calibrated single module data is requested. And want to update it once I have data for calibrated single modules. The reason for not waiting is because a user needs to use the updated simulator before this friday

Reply:
:+1: